app/models/manageiq/providers/hawkular/middleware_manager/middleware_server.rb

```diff
@@ -1,4 +1,11 @@
 module ManageIQ::Providers
   class Hawkular::MiddlewareManager::MiddlewareServer < MiddlewareServer
+    def hawkular?
+      product == 'Hawkular'
```

Comment:
If this is already of type `Hawkular::MiddlewareManager::MiddlewareServer` do we need to check if the product is `'Hawkular'`?  What is the alternative?